java/source/data.rst

```diff
@@ -15,7 +15,7 @@ Compare Vectors for Field Equality
     import org.apache.arrow.vector.compare.TypeEqualsVisitor;
     import org.apache.arrow.memory.RootAllocator;
 
-    RootAllocator rootAllocator = new RootAllocator(Long.MAX_VALUE);
+    RootAllocator rootAllocator = new RootAllocator();
```

Comment:
Can we fix all of these to be `BufferAllocator allocator = ...`?

Reply:
Changed